.github/workflows/e2e-tests.yaml

```diff
@@ -39,14 +43,11 @@ jobs:
       uses: loft-sh/setup-devspace@main
 
     - name: Deploy vcluster
-      run: devspace run deploy --skip-push
-
-    - name: Load built image into kind cluster
-      run: docker images | grep ghcr.io/loft-sh/loft-enterprise/dev-vcluster | head -n 1 | awk '{print $1":"$2}' | xargs kind load docker-image
+      run: devspace run deploy -p ${{ matrix.distribution}} -p kind-load --var K8S_REPLICAS=1 --skip-push
```

Comment:
I changed the replicas to one because I was hitting resource issues in the runner:
`FailedScheduling  ...  0/1 nodes are available: 1 Insufficient cpu.`
but I don't think this is ideal. Is it possible to increase CPUs for the runner?
If not, maybe creating a devspace profile with no CPU requests would be a better solution?

Reply:
Yeah I guess creating a different profile makes sense, testing the HA setup is definitely a good idea in the pipeline if that works.